content/zh/docs/ops/configuration/traffic-management/manage-mesh-certificates/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 管理网格内证书
+linktitle: 管理网格内证书
+description: 如何在您的网格内配置证书。
```

Comment:
```suggestion
description: 如何配置您网格内的证书。
```

Reply:
The suggested change seems better, thanks.